apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -435,9 +413,15 @@ data class Aarsoppgjoer(
     val avkortetYtelseAar: List<AvkortetYtelse> = emptyList(),
 ) {
     init {
-        check(ytelseFoerAvkorting.zipWithNext().all { it.first.periode.fom < it.second.periode.fom })
-        check(inntektsavkorting.zipWithNext().all { it.first.grunnlag.periode.fom < it.second.grunnlag.periode.fom })
-        check(avkortetYtelseAar.zipWithNext().all { it.first.periode.fom < it.second.periode.fom })
+        check(ytelseFoerAvkorting.zipWithNext().all { it.first.periode.fom < it.second.periode.fom }) {
```

Comment:
Tror ikke `check` kommer til å gi så gode feilmeldinger ut, ref diskusjon på utviklersync forrige gang: https://nav-it.slack.com/archives/C01PG1JM9D4/p1725447491563459

Reply:
Ja, det er nice å ha disse som `if (betingelseICheck) { throw InternfeilException("melding i check-body") }

Reply:
Will do. Disse sjekkene bør virkelig ikke slå ut i frontend da. Da har noe virkelig gått galt. Dette er mer støtte under utvikling som slår ut når man kjører unittester.